src/api/crypto/Seed.cpp

```diff
@@ -756,17 +754,11 @@ auto Seed::SetDefault(const Identifier& id) const noexcept -> bool
 
     const auto seedID = id.str();
 
-    auto exists = false;
-    for (const auto& [value, alias] : api_.Storage().SeedList()) {
-        if (value == seedID) {
-            exists = true;
-            break;
-        }
-    }
-
-    if (false == exists) {
+    if (std::none_of(
+            api_.Storage().SeedList().cbegin(),
+            api_.Storage().SeedList().cend(),
+            [&seedID](const auto& seed) { return seed.first == seedID; })) {
```

Comment:
If i weel understood, we are doing check on each element of a collection while before we were skipping some of the iteration if we found some value, so maybe sth like find_first_of would leave the same logic as before ?

Reply:
https://en.cppreference.com/w/cpp/algorithm/all_any_none_of
Below there is Truth table for it. If it finds first matching element it will stop iteration